### ISSUE TITLE
`mlflow ui ` 

### ISSUE THREAD

**Issue by medch7** [mlflow/mlflow#3806]

**Comment by harupy** [comment]
What happens if you run `mlflow` in the terminal?

**Comment by medch7** [comment]
This is the output that I get , but It doesn't redirect me to ui 
`[746] [INFO] Starting gunicorn 20.0.4`
`[746] [INFO] Listening at: http://127.0.0.1:5000 (746)`
`[746] [INFO] Using worker: sync`
`[749] [INFO] Booting worker with pid: 749`


I should clarify that `mlflow ui` works fine on a local machine but the problem is on an online hosted  jupyterhub environment.

**Comment by dmatrix** [comment]
@medch7 Does it work on jupyter lab on your local host when launched from the directory where you have your local `mlruns` directory?

**Comment by medch7** [comment]
yes it works fine on my local host, what I need to know is what should I do when I'm working with a jupytehub environment hosted on kubernetes for examples (ie: how to stock experiments in a server and then be able to access them via `mlflow ui `)

**Comment by nitesh31mishra** [comment]
Can you just checkout with the host and port. On which port your mlflow ui and Jupyter is running and whether you are configuring the mlflowui to run on the host ip.
